tlv/primitive.go

```diff
@@ -73,6 +73,15 @@ func EUint8(w io.Writer, val interface{}, buf *[8]byte) error {
 	return ErrTypeForEncoding{val, "uint8"}
 }
 
+// EUint8T encodes a uint8 val to the provided io.Writer. This method is exposed
```

Comment:
Commit message should have an explanation why these methods are needed. Not obvious to me.